ci_build_images/debian-release.Dockerfile

```diff
@@ -8,7 +8,7 @@ FROM "$BASE_IMAGE"
 LABEL maintainer="MariaDB Buildbot maintainers"
 
 # This will make apt-get install without question
-ARG DEBIAN_FRONTEND=noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
Do we want this setting to persist in the built image?
We may need to get an interactive shell into the container and do some package operations where a human prompt would be useful. 

Maybe switching it back to default at the end of the block where its needed?
Or use it specifically for the commands that can cause problems and -y is just not enough.

Reply:
Good comment and I agree with https://serverfault.com/a/797318 and probably 2/ (using it in the RUN call) is the cleaner way.